src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -869,19 +872,114 @@ bool Limitations::is_split_supported(const std::shared_ptr<ov::Node>& node, bool
     return is_aligned;
 }
 
+bool Limitations::is_concat_supported(const std::shared_ptr<const ov::Node>& node) {
+    OPENVINO_ASSERT(node, "Concat node is empty!");
+    auto concat_node = std::dynamic_pointer_cast<const ngraph::opset9::Concat>(node);
+    const ov::Shape& output_shape = concat_node->get_output_shape(0);
+    auto axis = concat_node->get_axis();
+
+    return graph_utils::get_first_valuable_dim_id(output_shape) == axis;
```

Comment:
Looks like this check doesn't over all the cases, for example concat (1,1,2)x(1,1,2) - > (1, 2, 2)

Reply:
It seems to be ok in a such case